docs/index.md

```
@@ -0,0 +1,137 @@
+```json
+{
+  "status": "healthy",
+  "message": "API is running smoothly",
+  "timestamp": 1640995200,
+  "version": "1.0.0"
+}
+```
+
+## 🏗️ Project Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)